keyboards/tokyokeyboard/alix40/alix40.c

```diff
@@ -0,0 +1,22 @@
+#include "alix40.h"
```

Comment:
All alix40 `.c` and `.h` files need GPL compatible headers added

Reply:
Fixed.